test-case/multiple-pipeline.sh

```diff
@@ -35,6 +33,10 @@ OPT_PARM_lst['t']=1         OPT_VALUE_lst['t']="$TPLG"
 OPT_OPT_lst['c']='count'    OPT_DESC_lst['c']='test pipeline count'
 OPT_PARM_lst['c']=1         OPT_VALUE_lst['c']=4
 
+OPT_OPT_lst['f']='first'
+OPT_DESC_lst['f']='Fill either playback (p) or capture (c) first'
+OPT_PARM_lst['f']=1         OPT_VALUE_lst['f']='p'
+
```

Comment:
@marc-hb The option `f ` means first, that's good for now.  In fact, I want one more option for `all pipelines`,  then the `first` may not be proper.